plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -462,25 +462,93 @@ fu_uefi_capsule_plugin_load_config(FuPlugin *plugin, FuDevice *device)
 		fu_device_add_private_flag(device, FU_UEFI_DEVICE_FLAG_FALLBACK_TO_REMOVABLE_PATH);
 }
 
+static gboolean
+fu_uefi_capsule_plugin_is_esp_linux(FuVolume *esp, GError **error)
+{
+	g_autofree gchar *basenames_str = NULL;
+	g_autofree gchar *mount_point = fu_volume_get_mount_point(esp);
+	g_autoptr(GPtrArray) files = NULL;
+	const gchar *const basenames[] = {"grubaa64.efi",
+					  "grubia32.efi",
```

Comment:
We know the suffix for the architecture at compile time. Why not use that instead of hard coding all possible architectures?

Reply:
Good idea! fixed.